docs/ops.html

```diff
@@ -3956,25 +3956,31 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
     While the cluster is still in migration mode, it is possible to revert to ZK mode. In order to do this:
     <ol>
       <li>
-        For each KRaft broker:
+        If you have brokers in KRaft mode, for each one:
```

Comment:
Since the definition of `brokers in KRaft mode` is not clear during migration process, maybe we can say this:

`If you have brokers in KRaft mode (i.e. no <code>zookeeper.connect</code> is configured), for each one:`